doc/Command_Index_User.tex

```diff
@@ -394,6 +396,11 @@ \subsubsection{Score Font}
   & \texttt{parmesan} & Use the Parmesan font.\\
 \end{argtable}
 
+\macroname{\textbackslash gresetgregoriofontscaled}{[\optional{\#1}]\{\#2\}\{\#3\}}{gregoriotex-main.tex}
+
+This function is the same as above, with a third argument to scale the font. The fonts shipped with Gregorio do not need to use this function, but some custom fonts do. Note that you cannot use this to scale glyphs up or down, as they would not be placed correctly on the staff.
+
+The two first arguments are the same as \texttt{\textbackslash gresetgregoriofont}. The third argument is an integer representing the scaling factor, where the one used by \texttt{\textbackslash gresetgregoriofont} is 100000.
```

Comment:
I'd be inclined to repeat the argtable in general, but so long as the reference is to the command described immediately before (as it is in this case), I think this is acceptable.